rainbows.rb

```diff
@@ -27,3 +31,21 @@ def mix_colors(color1,color2)
 
 new_color = mix_colors(red,green)
 puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
+
+# Let's create a rainbow of colors as an array of hashes, where each hash contains the name of the color (key) 
+# and labeled RGB values (value).
+
+
+
+new_color = mix_colors(red,green)
+puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
+
+
+
+def print_rainbow(array)
+  array.length.times do |iteration|
+    puts array[iteration].keys.join.upcase
+  end
+end
```

Comment:
This is ok, but won't allow us to colorize the words later. Even if we don't do the colorizing, it would have been nice to write code that would allow adding that easily later.